Shell/Shell.h

```diff
@@ -174,6 +174,26 @@ class Shell : public Core::Object {
 
 #undef __ENUMERATE_SHELL_BUILTIN
 
+    constexpr static const char* BuiltinNames[] = {
```

Comment:
`builtin_names`

Reply:
I had taken to doing PascalCase for constexpr members...sure.